vitest.config.ts

```diff
@@ -10,6 +11,10 @@ process.env = {
 	...readCanisterIds({ prefix: 'VITE_' })
 };
 
+configure({
```

Comment:
What about adding this to `vitest.setup` instead?

Reply:
indeed, it was my mistake, i changed it to setup